lib/solidus_shipstation/api/batch_syncer.rb

```diff
@@ -54,6 +54,7 @@ def post_sync(shipstation_order, shipments)
         shipment.update_columns(
           shipstation_synced_at: Time.zone.now,
           shipstation_order_id: shipstation_order['orderId'],
+          store_id: shipstation_order['advancedOptions']['storeId']
```

Comment:
What if it's null / not passed?

Reply:
I think we always get the store ID from `OrderSelectorSerializer`, but it wouldn't hurt to add a check